README.md

```diff
@@ -47,15 +47,15 @@ Motion::Project::App.setup do |app|
 end
 ```
 
-You will find the Project ID in your [project settings](https://phraseapp.com/projects). Generate Access Tokens in your [profile settings](https://phraseapp.com/settings/oauth_access_tokens).
+You will find the Project ID in your [project settings](https://phrase.com/projects). Generate Access Tokens in your [profile settings](https://phrase.com/settings/oauth_access_tokens).
```

Comment:
Even though this is already merged: I noticed that both URLs don't work and need the `app.` prefix.